shared/settings/advanced/index.tsx

```diff
@@ -90,54 +91,61 @@ const LockdownCheckbox = (props: Props) => {
   )
 }
 
-const Advanced = (props: Props) => (
-  <Kb.ScrollView style={styles.scrollview}>
-    <Kb.Box style={styles.advancedContainer}>
-      {props.settingLockdownMode && (
-        <Kb.Box style={styles.progressContainer}>
-          <Kb.ProgressIndicator />
-        </Kb.Box>
-      )}
-      <Kb.Box style={styles.checkboxContainer}>
-        <LockdownCheckbox {...props} />
-      </Kb.Box>
-      {!!props.setLockdownModeError && (
-        <Kb.Text type="BodySmall" style={styles.error}>
-          {props.setLockdownModeError}
-        </Kb.Text>
-      )}
-      {!props.hasRandomPW && (
+const Advanced = (props: Props) => {
+  const {load} = props
+  React.useEffect(() => {
+    load()
+    // eslint-disable-next-line
```

Comment:
Could we just disable the lint rule?

Reply:
I think the rule is actually good but we need this use case sometimes